src/main/resources/db/changelog/data/financialTransaction.csv

```diff
@@ -0,0 +1,9 @@
+id,wallet_id,transaction_type,amount,transaction_date,description
```

Comment:
Since the .csv files contain the data for the tables, you can name them the same as the tables.

financialTransaction.csv -> financial_transactions.csv
wallet.csv -> wallets.csv